modin/experimental/xgboost/xgboost.py

```diff
@@ -97,11 +272,7 @@ class Booster(xgb.Booster):
     def __init__(self, params=None, cache=(), model_file=None):  # noqa: MD01
         super(Booster, self).__init__(params=params, cache=cache, model_file=model_file)
 
-    def predict(
-        self,
-        data: DMatrix,
-        **kwargs,
-    ):
+    def predict(self, data: DMatrix, **kwargs):
```

Comment:
What's the need for this change?

Reply:
this was changed

Reply:
What do you mean? Why're these formatting changes made?

Reply:
I think that this changes doing `black`

Reply:
I think, that we should revert unnecessary changes.

Reply:
This was reset